### ISSUE TITLE
Scatter plot improvements

### ISSUE THREAD

**Issue by vianamp** [allen-cell-animated/nucmorph-colorizer#200]
- Download scatter plot as PDF
- Change point size and labels font size in the scatter plot

**Comment by ShrimpCryptid** [comment]
@vianamp can you clarify what needs the PDF format meets? 
Do you need the points as an SVG, or is something rasterized (PNG) okay?

**Comment by vianamp** [comment]
I was thinking of a use case where I want to stick the plot in a presentation. If the PNG quality is good then PNG should be fine.

**Comment by ShrimpCryptid** [comment]
Okay, gotcha, thanks for the clarification. PNG export should definitely be achievable. Is there a resolution (or range of resolutions) that feels appropriate for a graph?

I found some documentation from the library we're using for graphing and I'm including it in this thread for my own reference: https://plotly.com/javascript/static-image-export/

**Comment by vianamp** [comment]
We use 150 dpi minimally when we create a plot using matplotlib.

**Comment by ShrimpCryptid** [comment]
Is that with the default matplotlib graph size? (6.4 x 4.8?)

**Comment by vianamp** [comment]
mm, I am confused. The plot size does not really matter as long as you set the dpi. It will adjust the size accordingly to produce the specified dpi.

**Comment by ShrimpCryptid** [comment]
https://stackoverflow.com/a/47639545
I found this explanation of the relationship between figure size, dpi, and pixel size! What I'm looking for is a pixel size, rather than a raw dpi. If you don't have a target size in pixels off the top of your head, that's fine.

**Comment by toloudis** [comment]
How important is this for the March 31 deliverables?

**Comment by toloudis** [comment]
summary: plotly supports exporting a PNG.  We know you want 150 dpi minimum but we need to know the desired output image size.

**Comment by toloudis** [comment]
UX for font sizes in plot: plot section in settings tab, or settings in scatter plot tab?

**Comment by vianamp** [comment]
I don't think it is important for deliverable. Just a general feature that would be nice to have,

**Comment by ShrimpCryptid** [comment]
Note: Some limitations on how customizable we can make Plotly. This should only be used for internal presentations most likely and not for publications...
